.travis.yml

```diff
@@ -20,7 +20,7 @@ script:
 after_success:
   - if [ "$TRAVIS_REPO_SLUG" == "CodeForPoznan/pah-fm" ] && \
```

Comment:
I think that this bash-if stuff in here is pretty flawed, because it uses `\` for inlining the if cases but the block string in here will not perserve newlines currently, which gives us a one long string full of `&& \ [ ...` which breaks bash.
```
pip install yq
yq -r .after_success[0] .travis.yml
if [ "$TRAVIS_REPO_SLUG" == "CodeForPoznan/pah-fm" ] && \ [ "$TRAVIS_BRANCH" == "develop" ] && \ [ "$TRAVIS_PULL_REQUEST" = false ]; then docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD; cd backend; docker build -f Dockerfile -t codeforpoznan/pah-fm-backend:latest .; docker push codeforpoznan/pah-fm-backend; cd ../frontend; docker build -f Dockerfile -t codeforpoznan/pah-fm-frontend:latest .; docker push codeforpoznan/pah-fm-frontend; cd ..; fi
```
Here's my proposal:
```suggestion
  - |
    if [ "$TRAVIS_REPO_SLUG" == "CodeForPoznan/pah-fm" ] &&
       [ "$TRAVIS_BRANCH" == "develop" ] &&
       [ "$TRAVIS_PULL_REQUEST" = false ];
    then
      docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD

      docker build -t codeforpoznan/pah-fm-backend:latest ./backend
      docker push codeforpoznan/pah-fm-backend

      docker build -t codeforpoznan/pah-fm-frontend:latest ./frontend
      docker push codeforpoznan/pah-fm-frontend
    fi
```

And a way of checking that it's a correct approach:
```
pip install yq
yq -r .after_success[0] .travis.yml
<correct bash output spanning across several lines>
```